app/webhook_test.go

```diff
@@ -120,31 +120,29 @@ func TestCreateIncomingWebhookForChannel(t *testing.T) {
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
-			assert := assert.New(t)
-
 			th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableIncomingWebhooks = tc.EnableIncomingHooks })
 			th.App.UpdateConfig(func(cfg *model.Config) {
 				*cfg.ServiceSettings.EnablePostUsernameOverride = tc.EnablePostUsernameOverride
 			})
 			th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnablePostIconOverride = tc.EnablePostIconOverride })
 
 			createdHook, err := th.App.CreateIncomingWebhookForChannel(th.BasicUser.Id, th.BasicChannel, &tc.IncomingWebhook)
-			if tc.ExpectedError && err == nil {
-				t.Fatal("should have failed")
-			} else if !tc.ExpectedError && err != nil {
-				t.Fatalf("should not have failed: %v", err.Error())
+			if tc.ExpectedError {
+				assert.NotNil(t, err, "should have failed")
```

Comment:
As we are replacing `t.Fatal` here, shouldn't we use `require` instead of `assert` to stop the execution?

Reply:
Yes, you are right

Reply:
fixed